index.js

```diff
@@ -29,24 +27,68 @@ app.use( "*", cloudinaryConfig );
 
 mongoClient.connect(url, {useNewUrlParser : true}, function(err, client){
     if(err) throw err    
-    app.locals.db = client.db('markit')   
+    app.locals.db = client.db('markit')
+    app.locals.db.createCollection('users')   
     console.log("database connected!")   
 })
 
 app.use(bodyParser.json());
 
 app.use(express.static('./public'));
+app.use(bodyParser.urlencoded({ 
+    extended: true
+})); 
+
+ 
+
+
+app.post('/signup' ,function(req,res){
+    var name = req.body.name;
+    var username = req.body.username;
+	var email = req.body.email;
+    var password = req.body.password;				
+
+	var data = {
+        "name":name,
+        "username":username,
+		"email":email,
+        "password": password
+    }
+    req.app.locals.db.collection('users').insertOne(data, function(err, collection) {
+    if(err) throw err
+    console.log("Record inserted Successfully");
+     
+    });
+    res.redirect('/index.html'); 
+})
+
+app.post('/login', function (req, res) {
+
+    req.app.locals.db.collection('users').findOne({
+        username: req.body.log_username,
```

Comment:
You are using req.body.log_username here and you have named your name attribute as 'username' only how is that going to work ?

Reply:
ok